nengo_dl/tests/test_neurons.py

```diff
@@ -5,12 +5,12 @@
 import pytest
 
 from nengo_dl import (
-    LeakyReLU,
-    SpikingLeakyReLU,
-    SoftLIFRate,
     config,
     dists,
+    LeakyReLU,
     neuron_builders,
+    SoftLIFRate,
+    SpikingLeakyReLU,
```

Comment:
Sorting the imports based on the standard string sorting (i.e. capitals before lower case) feels more natural to me, but I'm not going to die on that hill 😉.

Reply:
Yeah I feel that. Will switch back!